docs/source/operations/fabric_networkyaml.md

```diff
@@ -272,6 +291,7 @@ The snapshot of an organization field with sample values is below
       type: orderer
       external_url_suffix: org1ambassador.blockchaincloudpoc.com
       org_status: new
+      orderer_org: supplychain # Name of the organization that provides the ordering service
```

Comment:
[nitpick] The example here is of `orderer` organization and the description for this field is that it is only to be added for `peer` organizations.

Reply:
Fixed